.gitignore

```diff
@@ -100,3 +100,4 @@ venv/*
 tmp/*
 */amici_models/*
 *.txt
+test/doc/example/tmp/benchmark-models/
```

Comment:
this directory should not actually be created when the tests are run from the base directory, because the menchmark-models are downloaded do ``pypesto/doc/example/tmp/benchmark-models``, not the test subfolder. but can't be wrong to catch this one as well.

Reply:
I consistently see this directory ...

Reply:
might be that you run tests differently (maybe ``python -m pytest`` from the test subdirectory? Anyway, then let' include it here.